cmd/utils/flags.go

```diff
@@ -659,6 +661,16 @@ func MakeSystemNode(name, version string, extra []byte, ctx *cli.Context) *node.
 	// Configure the Ethereum service
 	accman := MakeAccountManager(ctx)
 
+	// initialise new random number generator
+	rand := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
I don't think it's really that important. Why do you think we'd need additional entropy?